hrui/js/src/civihr-popup/civihr-popup.js

```diff
@@ -4,14 +4,61 @@ Prevent Popups to overflow tables.
 (function ($) {
   'use strict';
 
-  // .btn-slide is a continer of popup and a button triggering it
-  $('.btn-slide').attrchange(function (e) {
-    var $body = $('body');
+  $('body').on('click', '.btn-slide', function (e) {
     var $button = $(this);
-    var $closestEntity, $popup, buttonOffset;
+    var $body = $('body');
```

Comment:
I think this can go outside this function and then you can also do `$body.on('click'...`